### PR TITLE
Create ProLiant_DL380_Gen10.yaml

### PR DESCRIPTION
**Pull request by bbrk364**
adding new type for DL380 Gen10

### CODE REVIEW

**Comment by weidi** [comment]
This is in #769

**Comment by florianschendel** [comment]
@weidi You have to commit your changes into your created branch

File: device-types/HPE/ProLiant_DL380_Gen10.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - name: Rear Optional Serial
+    type: de-9
+  - name: Rear VGA
+    type: other
+  - name: Front USB1
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/HPE/ProLiant_DL380_Gen10.yaml:21]: This is not the intended use of USB ports.   They should not be modelled in this fashion unless they are truely used as a console port.

File: device-types/HPE/ProLiant_DL380_Gen10.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+device-bays:
+  - name: FlexibleLOM2 
+  - name: Pri Riser/PCIe 1
+  - name: Pri Riser/PCIe 2
+  - name: Pri Riser/PCIe 3
+  - name: 2nd Riser/PCIe 4
+  - name: 2nd Riser/PCIe 5
+  - name: 2nd Riser/PCIe 6
+  - name: 3nd Riser/PCIe 7
+  - name: 3nd Riser/PCIe 7
```
Comment (DanSheps) [inline device-types/HPE/ProLiant_DL380_Gen10.yaml:54]: This is not the intended use of device bays

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Pull request closed without merging**